samples/Quickstart/src/Filters/QuickstartFilter.cs

```diff
@@ -1,6 +1,16 @@
-﻿using Azure.Health.DataServices.Clients;
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Health.DataServices.Clients;
+using Azure.Health.DataServices.Clients.Headers;
```

Comment:
Remove reference to custom headers...RestBinding will do this.

Reply:
Removed as suggested. please verify.